src/Handlers/SuppressHandler.php

```diff
@@ -39,13 +41,39 @@ class SuppressHandler implements AfterClassLikeVisitInterface
         ],
     ];
 
+    /**
+     * @var array<string, list<class-string>>
+     */
+    private const BY_NAMESPACE = [
+        'PropertyNotSetInConstructor' => [
+            'App\Jobs',
```

Comment:
hm, what about apps with custom namespaces? https://laravel.com/docs/5.0/structure#namespacing-your-application

really annoying that laravel jobs don't have an interface to implement and are just expected to have a `handle` method... 🤔 💭 I wonder if that is something that could be changed upstream

Reply:
> hm, what about apps with custom namespaces? https://laravel.com/docs/5.0/structure#namespacing-your-application

I didn't know about them, but the recent Laravel Docs (6.0+) don't seem to mention that anymore. Possibly a deprecated feature?

> really annoying that laravel jobs don't have an interface to implement and are just expected to have a handle method... thinking thought_balloon I wonder if that is something that could be changed upstream

Yeah, that would be a good idea, but I'm not sure what is the best way to propose this. Possibly via https://github.com/laravel/ideas first?

Reply:
>I didn't know about them, but the recent Laravel Docs (6.0+) don't seem to mention that anymore. Possibly a deprecated feature?

Ah, I guess so! The majority of my apps have all been around since 5.x and have been upgraded over the years, but still have a custom namespace.

----

I don't think custom namespaces should block this PR though!

Reply:
PS: The `app:name` command was removed in Laravel 6.0: https://github.com/laravel/framework/pull/27575